sdk/eventhub/azure-eventhub/TROUBLESHOOTING.md

```diff
@@ -96,19 +107,19 @@ Further reading:
 
 #### Adding "TransportType=AmqpWebSockets"
 
-To use web sockets, pass in a kwarg `transport_type = TransportType.AmqpOverWebsocket` during client creation.
+To use web sockets, pass in a kwarg `transport_type = TransportType.AmqpOverWebsocket` during client [creation][WebsocketConfig].
 
 #### Adding "Authentication=Managed Identity"
 
-To authenticate with Managed Identity, see the sample [client_identity_authentication.py][PublishEventsWithAzureIdentity].
+To authenticate with Managed Identity, see the [sample](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/eventhub/azure-eventhub/samples/sync_samples/client_identity_authentication.py).
 
-For more information about the `Azure.Identity` library, check out our [Authentication and the Azure SDK][AuthenticationAndTheAzureSDK] blog post.
+For more information about the `azure.identity` library, check out our [Authentication and the Azure SDK][AuthenticationAndTheAzureSDK] blog post.
 
 ## Enable and configure logging
 
 The Azure SDK for Python offers a consistent logging story to help troubleshoot application errors and expedite their resolution.  The logs produced will capture the flow of an application before reaching the terminal state to help locate the root issue.
 
-This library uses the standard [Logging] library for logging
+This library uses the standard [Logging] library for logging and an example to set up debug logging with transport level logging is [here](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/eventhub/azure-eventhub#logging). 
```

Comment:
We may want to copy that entire section over here in addition to the linking. It'll be easier to copy paste from here.